specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+This is the specified format for providing data in the form of key-value pairs, which are also known as name-value pairs, attribute-value pairs, and field-value pairs in other contexts. 
+
+This format is often used in place of enumerating data which would be inherently sparse and/or without predetermined keys (attributes) into separate columns. To faciliate parsing, this format does not support nesting.
+
+## Attribute ID
+
+KeyValueFormat
+
+## Attribute Name
+
+Key-Value Format
+
+## Description
+
+Rules and formatting requirements for columns appearing in billing data which convey data as key-value pairs.
+
+## Requirements
+
+* Columns subject to Key-Value Format MUST be a serialized JSON string, consistent with the ECMA-404 definition of an object.
+* Keys MUST be unique within an object.
+* Values MAY be a number, string, `true`, `false`, or `null`.
```

Comment:
Is it clear that this is referring to the Key and Value of each KV pair defined in this column? Line 23 for example could get confused with the overall value of the cell? (I could be misreading that)